app/models/skin.rb

```diff
@@ -127,6 +128,17 @@ def clean_css
   scope :unapproved_skins, where(:public => true, :official => false, :rejected => false)
   scope :rejected_skins, where(:public => true, :official => false, :rejected => true)
   scope :site_skins, where(:type => nil)
+  scope :wizard_site_skins, where("type IS NULL AND (
```

Comment:
This is only used for the after task at the moment, but it seems potentially useful include in the model. If others disagree, I'll move it.